### PR TITLE
(PC-12196) feat(privacy): Add refuse button to modal

### PR DESCRIPTION
**Pull request by BenStirrup**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12196

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         | image |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/profile/pages/ConsentSettings.tsx [hunk 1]
```diff
@@ -31,6 +31,11 @@ export const ConsentSettings: FunctionComponent<Props> = ({ route }) => {
 
   useEffect(() => {
     storage.readObject('has_accepted_cookie').then((hasAcceptedCookie) => {
+      // If the user has navigated to this page from the privacy modal, we consider that
+      // as an implicit refusal, which they can change with the switch if they want to.
+      if (hasAcceptedCookie === null) {
+        storage.saveObject('has_accepted_cookie', false)
```
Comment (antoinewg) [inline src/features/profile/pages/ConsentSettings.tsx:37]: ah on est très conservateurs quand  même. C'est okey avec les POs ça ?

**Review by antoinewg — APPROVED** [review]

File: src/features/firstLogin/PrivacyPolicy/PrivacyPolicyModal.tsx [hunk 2]
```diff
@@ -75,13 +76,34 @@ export const PrivacyPolicyModal: FunctionComponent<Props> = ({
           {t`Tu peux modifier tes paramètres de confidentialité ici ou dans la page profil.`}
         </Typo.Caption>
       </SubDescription>
-      <ButtonPrimary title={t`Continuer`} onPress={onApproval} />
-      <Spacer.Column numberOfSpaces={2} />
-      <ButtonPrimaryWhite title={t`Paramètres de confidentialité`} onPress={goToConsentSettings} />
+      <CallToActionsContainer>
+        <StyleButtonSecondary title={t`Refuser`} onPress={onRefusal} textSize={getSpacing(3)} />
+        <Spacer.Column numberOfSpaces={3} />
+        <StyleButtonPrimary title={t`Autoriser`} onPress={onApproval} />
```
Comment (antoinewg) [inline src/features/firstLogin/PrivacyPolicy/PrivacyPolicyModal.tsx:82]: c'est normal que sur le screenshot, le 2 boutons n'est pas la même largeur ?

File: src/features/profile/pages/ConsentSettings.tsx (hunk 1 above)
Comment (BenStirrup) [reply src/features/profile/pages/ConsentSettings.tsx:37]: je check

File: src/features/firstLogin/PrivacyPolicy/PrivacyPolicyModal.tsx (hunk 2 above)
Comment (BenStirrup) [reply src/features/firstLogin/PrivacyPolicy/PrivacyPolicyModal.tsx:82]: yes

**Pull request merged**